pkg/cli/cliflags/flags.go

```diff
@@ -324,6 +324,15 @@ disabled by setting the environment variable COCKROACH_SKIP_KEY_PERMISSION_CHECK
 		Description: `Path to the CA key.`,
 	}
 
+	MaxOffset = FlagInfo{
+		Name: "max-offset",
+		Description: `
+Maximum allowed clock offset for the cluster. If observed clock offsets exceed
```

Comment:
This flag needs to be set identically across all nodes, right? If so, let's add something like this at the end:

> Note that, if this value is changed, every node in the cluster must be stopped and restarted with the identical value.